src/gov-action-contracts/governance/SetProposalThresholdsAndConstitutionHashAction.sol

```diff
@@ -0,0 +1,41 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity 0.8.16;
+
+import "../address-registries/L2AddressRegistry.sol";
+import "./SetProposalThresholdLib.sol";
```

Comment:
I don't think its worth the indirection here especially since we're not reusing this anywhere.
could just have an in-line internal method here (or even a solidity free function at the top of the file)